simple_network_sim/loaders.py

```diff
@@ -203,6 +203,75 @@ def readInfectionProbability(df: pd.DataFrame) -> Dict[int, float]:
     return probs
 
 
+def readRandomSeed(df: pd.DataFrame) -> int:
+    """
+    Transforms the dataframe from the data API into a bool usable inside the model
+
+    :param df: a dataframe containing the random seed
+    :return: a value of using the random seed as an int
+    """
+    if df is None:
+        return 0
+
+    assert len(df) == 1
+    assert df.columns == ["Value"]
+
+    for row in df.to_dict(orient="row"):
+        seed = row["Value"]
+
+        if not isinstance(seed, int):
+            raise ValueError("Seed must be an int")
+
+        if seed < 0:
+            raise ValueError("Seed must be positive")
+
+        return seed
+
+
+def readTrials(df: pd.DataFrame) -> int:
+    """
+    Transforms the dataframe from the data API into a bool int inside the model
+
+    :param df: The dataframe containing the number of trials
+    :return: the number of trials to run
+    """
+    assert len(df) == 1
+    assert df.columns == ["Value"]
+
+    for row in df.to_dict(orient="row"):
+        trials = row["Value"]
+
+        if not isinstance(trials, int):
+            raise ValueError("trials must be an int")
+
+        if trials < 1:
+            raise ValueError("trials must be > 0")
+
+        return trials
+
+
+def readStochasticMode(df: pd.DataFrame) -> bool:
```

Comment:
If you agree with the idea of having a table of trials, maybe weather each trial will be stochastic or not could be a column in the table. I don't know.

Reply:
So you're seeing this as a table, where each row has:
1) Stochastic or deterministic
2) Seed 
I think I agree with this.

Reply:
I'm trying to think of a way to keep using tables for these parameters, rather than single values.